_config.yml

```diff
@@ -42,21 +42,21 @@ sphinx:
       - _build
       - node_modules
       - theme/src
-    html_theme: furo
+    #html_theme: furo
```

Comment:
If we;re replacing `furo` with the napari theme, then I'd probably remove this line entirely instead of just commenting it out.

Reply:
Oh sure - I just don't think this is actually enabled right now in CI, but I could be wrong